src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topSide;
+    private double bottomSide;
+    private double lateralSide;
+    private double area;
+
+    public IsoscelesTrapezoid(String color, double topSide, double bottomSide, double lateralSide) {
+        super(color);
+        this.topSide = topSide;
+        this.bottomSide = bottomSide;
+        this.lateralSide = lateralSide;
+        this.area = calculateArea();
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables and pass them on for better code readability. Also, consider if the area needs to be a field, as it can be calculated on-demand when calling `getArea()`.